test/k8s-integration/main.go

```diff
@@ -352,6 +355,9 @@ func generateGCETestSkip(clusterVersion string) string {
 	if v.LessThan(apimachineryversion.MustParseSemantic("1.16.0")) {
 		skipString = skipString + "|volumeMode\\sshould\\snot\\smount\\s/\\smap\\sunused\\svolumes\\sin\\sa\\spod"
 	}
+	if platform == "windows" {
```

Comment:
Can you put this before or after the version checks instead of in the middle?

Reply:
fixed